blivet/formats/luks.py

```diff
@@ -130,7 +130,10 @@ def __init__(self, **kwargs):
         self._key_file = kwargs.get("key_file")
         self.escrow_cert = kwargs.get("escrow_cert")
         self.add_backup_passphrase = kwargs.get("add_backup_passphrase", False)
-        self.min_luks_entropy = kwargs.get("min_luks_entropy", 0)
+        self.min_luks_entropy = kwargs.get("min_luks_entropy", None)
```

Comment:
This is a change in behavior in Blivet, isn't it? It was defaulting to `0` and now it defaults to `luks_data.min_entropy`.

Reply:
But `luks_data.min_entropy` is set to `0` by default in Blivet.

Reply:
Oh! That explains a lot :smile: